base/interactiveutil.jl

```diff
@@ -678,8 +673,7 @@ download(url, filename)
     workspace()
 
 Replace the top-level module (`Main`) with a new one, providing a clean workspace. The
-previous `Main` module is made available as `LastMain`. A previously-loaded package can be
-accessed using a statement such as `using LastMain.Package`.
```

Comment:
How does one do this now? / What does `workspace()` do? / Should we add `empty!(Base. loaded_modules)`, and (re-)register the new modules with `register_root_module`? / Should we add a new function to do that (`Base.reset_require()` or argument to `workspace`) instead?

Reply:
I'll update the doc string. I think `workspace()` should *just* give you a new Main module. So `using X` in the new Main module will point to the already-loaded copy. To re-load packages there is `reload`.